### DISCUSSION TITLE
WalletConnect verification

### DISCUSSION THREAD

**Discussion by PeeCee45** [WalletConnect/walletconnect-monorepo#3337] [Developer Support] [answered]
My wallet connect v2 works well on localhost, but when uploaded on my site its only generates the key(uri) no other action follows......(i.e the web3modal doesn't appears.

i then assumed that maybe my project id and website isn't verified yet on the wallet connect cloud, which am just 20% sure of

The verification takes up to three business days after been published but i don't understand why mine is taking much time.

can someone please assist me on this

**Comment by glitch-txs** [comment]
You don't need verification to be able to use WalletConnect. Do you have any console errors? Are you on Next.js btw?

**Reply by PeeCee45** [reply]
Thanks for your response

this are the errors its keep showing on console:

browser.js:202 {context: 'core'}context: "core"[[Prototype]]: Object {context: 'core/relayer'} Error: WebSocket connection closed abnormally with code: 3000 (Authorization error: Unauthorized: origin not allowed)
    at n.value (provider.ts:118:9)
    at o.<anonymous> (provider.ts:150:62)
    at o.emit (events.js:153:5)
    at e.value (ws.ts:150:17)
    at e.onclose (ws.ts:141:36)
t.<computed> @ browser.js:202
(anonymous) @ browser.js:266
(anonymous) @ browser.js:266
(anonymous) @ ws.ts:187
o.emit @ events.js:153
value @ provider.ts:116
(anonymous) @ provider.ts:150
o.emit @ events.js:153
value @ ws.ts:150
e.onclose @ ws.ts:141



The funniest thing is that it doesn't show me this on the localhost
Here's my react code:

import SignClient from "@walletconnect/sign-client";
import { Web3Modal } from "@web3modal/standalone";
import { useEffect, useState } from "react";


const projectId = process.env.REACT_APP_PROJECT_ID;


const web3Modal = new Web3Modal({
  projectId: projectId,
  standaloneChains: ["eip155:1"],
});

export default function HomePage() {
  const [signClient, setSignClient] = useState(undefined);

  // 3. Initialize sign client
  async function onInitializeSignClient() {
    const client = await SignClient.init({
      projectId: projectId,
    });
    setSignClient(client);
  }

  // 4. Initiate connection and pass pairing uri to the modal
  async function onOpenModal() {
    if (signClient) {
      const namespaces = {
        eip155: {
          methods: ["eth_sendTransaction"],
          chains: ["eip155:1"],
          events: ["connect", "disconnect"],
        },
      };
      const { uri, approval } = await signClient.connect({
        requiredNamespaces: namespaces,
      });
      console.log(uri);
      if (uri) {
        try {
          await web3Modal.openModal({
            uri,
            standaloneChains: namespaces.eip155.chains,
          });
          console.log(namespaces.eip155.chains);
          await approval();
          console.log(22222);
          web3Modal.closeModal();
        } catch (error) {
          console.log("Eeee", error);
        }
      }
    }
  }

  useEffect(() => {
    if (!signClient) {
      onInitializeSignClient();
    }
  }, []);

  return signClient ? (
    <button onClick={onOpenModal}>Connect Wallet</button>
  ) : (
    "Initializing..."
  );
}

**Reply by glitch-txs** [reply, marked as answer]
Seems like an issue with the whitelist of your Cloud project. Make sure to add your domain in the project whitelist (it's where you created the project ID). If the issue persists try creating a new project please.

**Reply by PeeCee45** [reply]
Thanks so much, it's popping up now

**Reply by PeeCee45** [reply]
Thanks so much, it's popping up now
